recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
-{% set version = hdf5_version|default("1.10.5") %}
+{% set version = "1.10.6" %}
```

Comment:
Do not change this line. This branch builds both 1.10.5 and 1.10.6

Reply:
ah, the ros3 driver is only available from 1.10.6 onwards. how would i address that given the changes.

Reply:
You can use jinja2 if blocks or selectors on the version

Reply:
resolved this in meta, but not in install.bat or build.sh